main.py

```diff
@@ -97,7 +97,8 @@ def version():
     try:
         command_args = ["git", "describe", "--tags"]
         f = subprocess.Popen(command_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        return f.stdout.readline().decode(encoding="utf-8").rstrip()
+        stdout, stderr = f.communicate()
```

Comment:
There's [subprocess.check_output](https://docs.python.org/3/library/subprocess.html#subprocess.check_output) function which should be exactly what we need here. (it does Popen and communicate calls for you)